src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -304,6 +320,14 @@ public override string Name
             }
         }
 
+        public override string MetadataName
+        {
+            get
+            {   
+                return HasNameInMetadata ? _name : string.Empty;
```

Comment:
I was under the impression that empty string was an invalid metadata name for parameters (but `null` was legal).  It looks like this gets translated under the hood to `StringIdx(0)`:

https://github.com/dotnet/roslyn/blob/master/src/Compilers/Core/Portable/PEWriter/MetadataHeapsBuilder.cs#L237

I can't tell if that gets translated to an empty string or `null` though.